packages/vitest/src/integrations/chai/jest-expect.ts

```diff
@@ -297,50 +303,55 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
   })
   def(['toHaveReturned', 'toReturn'], function() {
     const spy = getSpy(this)
+    const spyName = spy.getMockName()
     const calledAndNotThrew = spy.called && !spy.results.some(([type]) => type === 'error')
     this.assert(
       calledAndNotThrew,
-      'expected spy to be successfully called at least once',
-      'expected spy to not be successfully called',
+      `expected spy "${spyName}" to be successfully called at least once`,
+      `expected spy "${spyName}" to not be successfully called`,
       calledAndNotThrew,
       !calledAndNotThrew,
     )
   })
   def(['toHaveReturnedTimes', 'toReturnTimes'], function(times: number) {
     const spy = getSpy(this)
+    const spyName = spy.getMockName()
     const successfullReturns = spy.results.reduce((success, [type]) => type === 'error' ? success : ++success, 0)
     this.assert(
       successfullReturns === times,
-      `expected spy to be successfully called ${times} times`,
-      `expected spy to not be successfully called ${times} times`,
+      `expected spy "${spyName}" to be successfully called ${times} times`,
+      `expected spy "${spyName}" to not be successfully called ${times} times`,
       `expected number of returns: ${times}`,
       `received number of returns: ${successfullReturns}`,
     )
   })
   def(['toHaveReturnedWith', 'toReturnWith'], function(value: any) {
     const spy = getSpy(this)
+    const spyName = spy.getMockName()
     const pass = spy.results.some(([type, result]) => type === 'ok' && asymmetricEquals(value, result))
     this.assert(
       pass,
-      'expected spy to be successfully called with #{exp}',
-      'expected spy to not be successfully called with #{exp}',
+      `expected spy "${spyName}" to be successfully called with #{exp}`,
+      `expected spy "${spyName}" to not be successfully called with #{exp}`,
       value,
     )
   })
   def(['toHaveLastReturnedWith', 'lastReturnedWith'], function(value: any) {
     const spy = getSpy(this)
+    const spyName = spy.getMockName()
     const lastResult = spy.returns[spy.returns.length - 1]
     const pass = asymmetricEquals(lastResult, value)
     this.assert(
       pass,
-      'expected last spy call to return #{exp}',
-      'expected last spy call to not return #{exp}',
+      `expected last "${spyName}" spy call to return #{exp}`,
+      `expected last "${spyName}" spy call to not return #{exp}`,
```

Comment:
this will be 
`expected last "mock" spy call to return`
and in `toHaveNthReturnedWith` you use:
`expected 1st spy "mock" call to return`

they both sound meh 👀 
do we even need `spy`?

Reply:
Agree, maybe it should be `expected 1st "mock" call to return...`

Reply:
Fixed